tests/trace_compiler/guards_collapse.rtf

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
+
+export $(env | grep '^YKT\|YKD')
```

Comment:
What's the point of this line?

Reply:
I thought it was required for the sub-process to inherit our env, but it appears that's not true.

b1e3327 kills it.

Reply:
(and makes 5c5dd11 redundant)